api/db/populate.js

```diff
@@ -48,9 +50,12 @@ const populateCollection = async () => {
     case 'friendships':
       await createFriendships(userArgs.v);
       break;
+    case 'comments':
+      await createComments(userArgs.v);
+      break;
     default:
       console.log(
-        `Unknown collection '${userArgs.c}', possible options are: [interests, users, events, attendants]`
+        `Unknown collection '${userArgs.c}', possible options are: [interests, users, events, attendants, friendships, comments]`
```

Comment:
This seems like a rebase issue. While we are at it, I think it would be better to have `comments` before `friendships`, as it is already the case when populating the database, elsewhere -- I think this is just a bit of re-ordering in a few places.